docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '2.1'
+services:
+  docs:
+    build:
```

Comment:
If we make the volume mount changes below, then we can just reference the official Yarn container image without a `Dockerfile`: https://hub.docker.com/r/yarnpkg/node-yarn/